### PR TITLE
Document captchas and blacklists.

### PR DESCRIPTION
**Pull request by fisx**
This both describes my ideas of captchas and how traffic data and blacklists could be managed.  It also aims at making it explicit what each of the mechanisms is worth.  This should be a useful read for project managers to manage customer expectations and argue in favor of sms-tans.

### CODE REVIEW

File: docs/sybil.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+attackers.  The hope is that many attackers in the wild still fall
+short of that threshold.  (Traffic analysis should help you to draw
+your own conclusions there.)
+
+We have deliberately chosen to not rely on google's recaptcha.  It
+collects IP addresses of all your users, and the javascript code that
+it loads into your browser is obfuscated and may generate additional
+data traces now or in the future.  Furthermore, even though it is
+probably slightly harder to crack than our built-in solution, there is
+on price difference for those who decide to counter Captcha security
```
Comment (ChristianSi) [inline docs/sybil.md]: s/on/no/

File: docs/sybil.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+between the two.
+
+
+## Looking for patterns in registration traffic and blacklists
+
+If, say, there are hundrets of emails creating accounts on the same
+afternoon, and they all have the form `anon<a-few-digits>@yahoo.com`,
+then something is probably going wrong.
+
+Thentos allows to download tables of the following form via rest
```
Comment (ChristianSi) [inline docs/sybil.md]: may at some point in the future allow

File: docs/sybil.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+The end-point supports content types text/csv and application/json;
+the analysis happens off-line with either office or scripting tools of
+your choice.
+
+If the analyst decides that there is a sybil attack, there two
+counter-measures are supported:
+
+    1. you can upload a blacklist of email addresses that are not
+       accepted for user registration.  registration attempts from
+       addresses on any list are silently dropped on the floor.  the
```
Comment (ChristianSi) [inline docs/sybil.md]: That could put a big burden on support (lots of people complaining "I registered but never got a confirmation email").

**Comment by fisx** [comment]
@rittermo: could you take a look, merge or object, and pass it on to PMs if it makes sense?  (not urgent.)

**Comment by rittermo** [comment]
I added a comment/question at the end of the document. I wasn't sure, if it makes more sense to comment in the doc or here. Sorry.

**Pull request merged**